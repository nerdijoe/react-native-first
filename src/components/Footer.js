import React from 'react'
import {
  View,
  Text
} from 'react-native'

const styles = {
  myFooter: {
    display: 'flex',
    // backgroundColor: '#F6F6F6',
    margin: 5,
    padding: 3,
    flexDirection: 'row',
    justifyContent: 'flex-end'
  },
  footerText: {
    fontSize: 7

  }
}

class Footer extends React.Component {
  render () {
    return (
      <View style={styles.myFooter}>
        <Text style={styles.footerText}>© 2017, クラロワ大好きです。</Text>
      </View>
    )
  }
}

export default Footer
