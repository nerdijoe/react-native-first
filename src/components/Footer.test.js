import React from 'react'
import renderer from 'react-test-renderer'

import Footer from './Footer'


describe ('Footer', () => {
  it('render without crashing', () => {
    const tree = renderer.create(<Footer />)

    expect(tree).toMatchSnapshot()
  })
})
