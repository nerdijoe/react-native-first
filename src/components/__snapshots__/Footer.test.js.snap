// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render without crashing 1`] = `
<View
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "flex-end",
      "margin": 5,
      "padding": 3,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 7,
      }
    }
  >
    © 2017, クラロワ大好きです。
  </Text>
</View>
`;
